src/requests/actions.jl

```diff
@@ -145,12 +143,12 @@ function expand_inline_func(x, id, server)
         end
         newtext = string(newtext, "\nend\n")
         tde = TextDocumentEdit(VersionedTextDocumentIdentifier(file._uri, file._version), TextEdit[TextEdit(Range(file, offset .+ (0:func.fullspan)), newtext)])
-        JSONRPCEndpoints.send_request(server.jr_endpoint, "workspace/applyEdit", ApplyWorkspaceEditParams(missing, WorkspaceEdit(nothing, TextDocumentEdit[tde])))
+        JSONRPC.send(conn, workspace_applyEdit_request_type, ApplyWorkspaceEditParams(missing, WorkspaceEdit(nothing, TextDocumentEdit[tde])))
```

Comment:
Unrelated but I think the first arg to WorkspaceEdit should be missing rather than nothing. Same thing a few lines above

Reply:
Best solved in a new PR, I'd say?